executor/api/grpc/seldon/server.go

```diff
@@ -56,9 +56,60 @@ func (g GrpcSeldonServer) SendFeedback(ctx context.Context, req *proto.Feedback)
 	return payloadToMessage(resPayload), nil
 }
 
+func (g GrpcSeldonServer) Metadata(ctx context.Context, req *proto.SeldonModelMetadataRequest) (*proto.SeldonModelMetadata, error) {
+	seldonPredictorProcess := predictor.NewPredictorProcess(ctx, g.Client, logf.Log.WithName("SeldonMessageRestClient"), g.ServerUrl, g.Namespace, grpc.CollectMetadata(ctx))
+	resPayload, err := seldonPredictorProcess.Metadata(g.predictor.Graph, req.GetName(), nil)
+	if err != nil {
+		return nil, err
+	}
+	return payloadToModelMetadata(resPayload), nil
+}
+
+func (g GrpcSeldonServer) GraphMetadata(ctx context.Context, req *empty.Empty) (*proto.SeldonGraphMetadata, error) {
+
+	seldonPredictorProcess := predictor.NewPredictorProcess(ctx, g.Client, logf.Log.WithName("SeldonMessageRestClient"), g.ServerUrl, g.Namespace, grpc.CollectMetadata(ctx))
+
+	graphMetadata, err := predictor.NewGraphMetadata(&seldonPredictorProcess, g.predictor)
```

Comment:
Would be good to keep consistency with how it's called in this GraphMetadata function to how it's called in the Metadata function above, as here it's called through a class function NewGraphMetadata in metadata.go and teh one above directly from the predictor process